content/usgs-packages/Getting_Started_USGS.Rmd

```diff
@@ -66,4 +66,10 @@ htmlTable(data.frame(Module, Description, Duration),
 
 ## Software requirements
 
-See [Before the workshop](/intro-curriculum/Before).
+See the [R installation instructions page](/installr) for how to install/upgrade R and RStudio, add GRAN to your settings, and install some basic packages. Then, execute these lines so that you have the most up-to-date version of the packages used in this course.
+
+```{r eval=FALSE}
+install.packages('dataRetrieval')
+install.packages('geoknife')
+install.packages('sbtools')
+```
```

Comment:
heh - with that change, that makes this PR competitive for shortest DS PR ever =)